### ISSUE TITLE
The drop rate of Durotar Tiger Fur is too high

### ISSUE THREAD

**Issue by Simp-N** [the-hyjal-project/bugtracker#216]
**Describe Your Issue**
The drop rate of quest item [Durotar Tiger Fur](https://classic.wowhead.com/item=4892/durotar-tiger-fur), for the quest [Practical Prey](https://classic.wowhead.com/quest=817/practical-prey) is currently set to 80%. I believe this is too high.

**Steps To Reproduce**
``SELECT * FROM creature_loot_template WHERE item=4892``

**Expected Behavior**
[Allakhazam Feb 2005](https://web.archive.org/web/20050224205619/http://wow.allakhazam.com/db/item.html?witem=4892): ~35%
[Allakhazam Mar 2008](https://web.archive.org/web/20080318131727/http://wow.allakhazam.com/db/item.html?witem=4892): ~34%
[Thottbot Jun 2006](https://web.archive.org/web/20070602033731/http://www.thottbot.com/i4892): ~34%
[Thottbot Jan 2008](https://web.archive.org/web/20081017164317/http://thottbot.com/i4892): ~42%
[Wowhead Classic](https://classic.wowhead.com/item=4892/durotar-tiger-fur#dropped-by): ~45%

Perhaps a reasonable drop rate is 40%, which is inbetween these values.

**Comment by T1ti** [comment]
Quest item drops on database sites are not accurate at all, because it will report the % of people that also don't have the quest active, heavily lowering the real droprate.

The best way to find the droprate is probably to kill about 50 mobs with the quest active

**Comment by Simp-N** [comment]
> 
> 
> Quest item drops on database sites are not accurate at all, because it will report the % of people that also don't have the quest active, heavily lowering the real droprate. The reason the droprate increases over time is because players are becoming smarter and more efficient, they explore and grind less and focus on clearing quests more
> I'd say the drop rate for this one is probably about 50/60%
> 
> The best way to find the droprate is probably to kill about 50 mobs with the quest active

Interesting. I've been under the impression that these databases compensated for this, and that the drop rate sometimes'd be off because of bugged reports or whatever. But looking at some drop rates, you may be right. This certainly complicates this. I will try and do some more research regarding database drop rates tomorrow. Thanks for bringing this up.

**Comment by petage** [comment]
> 
> 
> Quest item drops on database sites are not accurate at all, because it will report the % of people that also don't have the quest active, heavily lowering the real droprate. The reason the droprate increases over time is because players are becoming smarter and more efficient, they explore and grind less and focus on clearing quests more
> I'd say the drop rate for this one is probably about 50/60%
> 
> The best way to find the droprate is probably to kill about 50 mobs with the quest active

I agree with what you said, but manually collecting data from Classic WoW is time-consuming assuming you'd like to get reliable information with a large enough sample size. I'd say the Classic Wowhead, Allakhazam and Thottbot values are a good starting point. It would probably be best to use the Classic Wowhead values with an added percentage on top, for example 10-20% added. Of course this would have to be looked case-by-case. But then again, development time is valuable so this might not be an ideal solution. If development time is limited, I'd just use the Classic Wowhead values without adding anything.

Anyway, I've seen a lot of quest item drop rates blatantly set to 80%, even for drops that are supposed to be rare. For this reason, I hope to see the drop rates adjusted of quest items we report.

**Comment by Simp-N** [comment]
I guess the best thing I can do is simply look for quests who's drop rate seems off, gather and report numbers from different sites and have the developers and staff decide on the final drop rate if they too think it should change. Pretty much what I've been doing already except I don't suggest a number myself. I don't know the exact number they've went with in my previous reports.
